pkg/operators/localmanager/localmanager.go

```diff
@@ -618,6 +677,35 @@ func (l *localManagerTraceWrapper) PreStart(gadgetCtx operators.GadgetContext) e
 }
 
 func (l *localManagerTraceWrapper) Start(gadgetCtx operators.GadgetContext) error {
+	host := l.params.Get(Host).AsBool()
+	var containers []*containercollection.Container
+
+	if l.manager.igManager != nil {
+		containerSelector := containercollection.ContainerSelector{
+			Runtime: containercollection.RuntimeSelector{
+				ContainerName: l.params.Get(ContainerName).AsString(),
+			},
+		}
+
+		log.Debugf("add subscription to igManager")
+		containers = l.manager.igManager.Subscribe(
+			uuid.New().String(),
+			containerSelector,
+			func(event containercollection.PubSubEvent) {},
```

Comment:
I think we need to call emitContainersDatasourceEvent here instead of in the other subscription, otherwise an event for the same container could be generated twice (or a container missed) as both subscriptions aren't guaranteed to be synchronized.